app/controllers/expences_controller.rb

```diff
@@ -0,0 +1,76 @@
+class ExpencesController < ApplicationController
+  before_action :set_expence, only: %i[ show edit update destroy ]
+
+  # GET /expences or /expences.json
+  def index
+    @expences = Expence.where(predefined: true).or(Expence.with_user(current_user.id))
+  end
+
+  # GET /expences/1 or /expences/1.json
+  def show
+
+  end
+
+  # GET /expences/new
+  def new
+    @expence = Expence.new
+  end
+
+  # GET /expences/1/edit
+  def edit
+
```

Comment:
extra line here. Have you installed rubocop for your project?

Reply:
Yes, we do separate task for robocop check because it to much error in rails from beggining